OSS13 Server/Sources/World/Camera/Camera.cpp

```diff
@@ -253,6 +253,13 @@ void Camera::SetPosition(const Tile * const newTile) {
 	tile = newTile;
 }
 
+uf::vec3i Camera::ViewCoordsToWorldCoords(uf::vec3i viewCoords)
+{
+	//This const consist value of shift from camera position to first tile on view
+	static const uf::vec3i CAMERA_POS_ON_SCREEN = { 10, 10, 0 };
```

Comment:
You can use `getVisibleAreaSide() / 2 + 1` or `fov + Global::tilegrid::MIN_PADDING + 1` here, instead of magic constant with magic comment.